tests/primer/packages_to_lint.json

```diff
@@ -0,0 +1,53 @@
+{
+  "scikit-learn": {
+    "url": "https://github.com/scikit-learn/scikit-learn.git",
+    "branch": "main",
+    "directories": "sklearn"
+  },
+  "flask": {
+    "url": "https://github.com/pallets/flask.git",
+    "branch": "main",
+    "directories": "src/flask"
+  },
+  "keras": {
+    "url": "https://github.com/keras-team/keras.git",
+    "branch": "master",
+    "directories": "keras"
+  },
+  "sentry": {
+    "url": "https://github.com/getsentry/sentry.git",
+    "branch": "master",
+    "directories": "src tests"
+  },
+  "django": {
+    "url": "https://github.com/django/django.git",
+    "branch": "main",
+    "directories": "django tests"
+  },
+  "pandas": {
+    "url": "https://github.com/pandas-dev/pandas.git",
+    "branch": "master",
+    "directories": "pandas"
```

Comment:
Let's exclude the `tests` folder within the `pandas` directory.

Reply:
How would you exclude the test directory without removing pandas entirely ?

Reply:
Can we use `--ignore-patterns` in the `args`?

Reply:
b0e2c331f6ce7c436c050ec14d15a05a3af6dc7f

I don't think we can ignore by directory with this pattern so I used ``test_``